share/completions/s3cmd.fish

```diff
@@ -0,0 +1,178 @@
+# s3cmd
+
+function __s3cmd_is_remote_path
+	commandline -pct | string match -r -- "^s3://.*"
```

Comment:
You'll usually want to use the "-q" flag for `string match`.

It's okay here since this is only used as a complete condition, which has its output already silenced.